ledger/sandbox-on-x/reference.conf

```diff
@@ -327,17 +330,22 @@ ledger {
         # JWKS URL for jwt-rs-256-jwks algorithm
         # url = "https://localhost/jwks.json"
 
+        # TODO config rationalization: convert to 'leeway'
         #leeway-options {
         #  # Define the default window in seconds in which the Not Before, Issued At and Expires At Claims will still be valid.
+        #  # TODO config rationalization: Convert to 'leeway = 500 seconds'
         #  leeway = 500
```

Comment:
What about converting this `leeway` to `default`?

Reply:
```suggestion
        #  default = 500
```

Reply:
I did think about it, I can see downside to both of these
`leeway.leeway` looks odd
`leeway.default` doesn't map 1-1 to the underlying 3rd party library and you need to explain in text that `default` means `leeway`